src/components/ebay-toggle-button-group/toggle-button-group.stories.js

```diff
@@ -0,0 +1,123 @@
+import { tagToString } from "../../../.storybook/storybook-code-source";
+import { addRenderBodies } from "../../../.storybook/utils";
+import readme from "./README.md";
+import component from "./index.marko";
+import withIconsTemplate from "./examples/icons.marko";
+import withIconsCode from "./examples/icons.marko?raw";
+import withDefaultTemplate from "./examples/withDefault.marko";
+import withDefaultCode from "./examples/withDefault.marko?raw";
+import controlledTemplate from "./examples/controlled.marko";
+import controlledCode from "./examples/controlled.marko?raw";
+
+const Template = (args) => ({
+    input: addRenderBodies(args),
+});
+
+export default {
+    title: "buttons/ebay-toggle-button-group",
+    component,
+    parameters: {
+        docs: {
+            description: {
+                component: readme,
+            },
+        },
+    },
+    argTypes: {
+        radio: {
```

Comment:
I agree (and i missed this part where radio is a boolean)
In general we want to have more explicit options in ebayui, even if we have only two variabts
Like variant=radio instead of radio=true

This is so if things change we have less breaking changes. 
But in this case we need variant since there are three.

Reply:
Ok, I called the third option `single` but I'm not super happy with that name